hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -287,7 +287,7 @@ def _create_data_matrix_lines(self, data_matrix):
                     unit_amount=data_matrix[item]['unit_amount']
                 ))
             self.clean_timesheets(data_matrix[item]['timesheets'])
-        return lines
+        return lines.sorted(key=lambda l: l.date)
```

Comment:
and also project, perhaps? :thinking:

Reply:
and also task hahahaha

Reply:
I'd extract the key to a function for extending

Reply:
Now `test_6` is angry because of the sorting :thinking: